service/insights_v3.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	. "github.com/talbright/go-desk/types"
+	. "github.com/talbright/go-desk/resource"
+	"net/http"
+)
+
+type InsightsV3Service struct {
+	client *Client
+}
+
+func (s *InsightsV3Service) Report(query *InsightsV3Query) (*InsightsV3Report, *http.Response, error) {
```

Comment:
Probably not critical to write an integration test for this, since #13 covers it, but would still be good to get a unit test for ```report.go``` mentioned below.